jdaviz/app.py

```diff
@@ -2286,17 +2286,24 @@ def _on_new_viewer(self, msg, vid=None, name=None, add_layers_to_viewer=False):
             msg.cls, data=msg.data, show=False)
         viewer.figure_widget.layout.height = '100%'
 
+        linked_by_wcs = self._link_type.lower() == 'wcs'
+
         if hasattr(viewer.state, 'linked_by_wcs'):
             orientation_plugin = self._jdaviz_helper.plugins.get('Orientation', None)
             if orientation_plugin is not None:
-                viewer.state.linked_by_wcs = orientation_plugin.link_type.selected == 'WCS'
+                linked_by_wcs = orientation_plugin.link_type.selected == 'WCS'
             elif len(self._viewer_store):
                 # The plugin would only not exist for instances of Imviz where the user has
                 # intentionally removed the Orientation plugin, but in that case we will
                 # adopt "linked_by_wcs" from the first (assuming all are the same)
                 # NOTE: deleting the default viewer is forbidden both by API and UI, but if
                 # for some reason that was the case here, linked_by_wcs will default to False
-                viewer.state.linked_by_wcs = list(self._viewer_store.values())[0].state.linked_by_wcs  # noqa
+                linked_by_wcs = self._jdaviz_helper.default_viewer.state.linked_by_wcs
```

Comment:
Ooops forgot about Kyle's changes. My bad.
```suggestion
                linked_by_wcs = self._jdaviz_helper.default_viewer._obj.state.linked_by_wcs
```